docs/data/charts/zoom-and-pan/ZoomControlled.js

```diff
@@ -30,7 +28,10 @@ export default function ZoomControlled() {
           },
         ]}
       />
-    </div>
+      <Button onClick={() => setZoom([{ axisId: 'my-x-axis', start: 0, end: 100 }])}>
```

Comment:
Maybe a different variant could make it more visible (that's just a thought) while reding the diff)

When looking at argos diff, I though you removed it. I suspect the placement being similar to an axis label, my brain just ignored it

![image](https://github.com/user-attachments/assets/b0eec580-5639-4d2e-b2b6-f2425271d4c5)